### ISSUE TITLE
polar plot legend bug

### ISSUE THREAD

**Issue by djkashtan** [plotly/plotly.js#95]
I get extraneous items in my legend if I have less than three datasets. I have a codepen example that demonstrates it.

Codepen example

If you change line 61 to:

var data = [trace1,trace2,trace3,trace4,trace5];
then you will see that the legend has no extraneous entries. I have been wrestling with this for a while but I am stumped :(

**Comment by etpinard** [comment]
From v1.0.0 release [notes](https://github.com/plotly/plotly.js/releases/tag/v1.0.0): 

> Polar charts are under-developed.

**Comment by TCAllen07** [comment]
Likely duplicate of [#592](https://github.com/plotly/plotly.js/issues/592)

**Comment by etpinard** [comment]
fixed in `scatterpolar` after #2200
